### ISSUE TITLE
Add a package maintainability score

### ISSUE THREAD

**Issue by vtsykun** [composer/packagist#1368]
When searching for a package it would be nice to see a rating/score index of how the package is actively maintained. For example, this value can be calculated based on the date of the last release and commit, the response time per issue and other GitHub API data. Currently, when searching for a package, the most popular package is not always good and this can leads to package selection errors, like here https://github.com/symfony/symfony/pull/49831
![Selection_1379](https://user-images.githubusercontent.com/21358010/228387633-92e65e54-e62f-4a7d-b2ff-725e0d79f867.png)

**Comment by Spomky** [comment]
I agree with this idea. It would be very nive to spot unmaintained or outdated libraries 👍🏽.

**Comment by zerai** [comment]
https://isitmaintained.com/#metrics

**Comment by 94noni** [comment]
perhaps related/linked with https://github.com/composer/packagist/issues/1320
maybe we can add the latest version and its release date

(of course the date does not mean anything when a package is battle tested and simple thus not require any further update)

**Comment by 94noni** [comment]
PR as discussion draft opened here https://github.com/composer/packagist/pull/1410
Feel free to comment :)
